bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=pulp-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=beta
 LABEL operators.operatorframework.io.bundle.channel.default.v1=beta
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.29.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.31.0
```

Comment:
Should we really need to bump the operator-sdk in the release phase?
It is breaking one of the pipeline jobs:
https://github.com/pulp/pulp-operator/actions/runs/7063522452/job/19229695096?pr=1166

If we need a new version of it, I think we can create a new PR.

Reply:
I think this change was introduced by the `make` step we have in the instructions. Was not a manually one.
Reverting it.